docs/execution-providers/ROCm-ExecutionProvider.md

```diff
@@ -19,14 +19,20 @@ The ROCm Execution Provider enables hardware accelerated computation on AMD ROCm
 
 ## Install
 
+**NOTE** Please make sure to install the proper version of Pytorch specified here [PyTorch Version](../install/#training-install-table-for-all-languages).
+
+For Nightly PyTorch builds please see [Pytorch home](https://pytorch.org/) and select ROCm as the Compute Platform.
+
 Pre-built binaries of ONNX Runtime with ROCm EP are published for most language bindings. Please reference [Install ORT](../install).
 
 ## Requirements
 
 
 |ONNX Runtime|ROCm|
 |---|---|
-|main|5.2.3|
+|main|5.4|
+|1.13|5.4|
+|1.13|5.3|
```

Comment:
Becuase of rocblas bug on ROCm5.3, we skip ROCm5.3 and use ROCm5.3.2 now. Could you add the minor version here?

Reply:
Yup no problem I can update this

Reply:
Updated